docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -41,12 +41,14 @@ image::images/additional-filters.png[Alerts table with Additional filters menu h
 [[group-alerts]]
 === Group alerts
 
-beta:[] You can group alerts by rule name, host name, user name, source IP address, or any other field. Select *Group alerts by*, then select an option or *Custom field* to specify a different field.
+You can group alerts by rule name, user name, host name, source IP address, or any other field. Select *Group alerts by*, then select an option or *Custom field* to specify a different field.
+
+Select up to three fields for grouping alerts. The groups will nest in the order you selected them.
 
 [role="screenshot"]
 image::images/group-alerts.png[Alerts table with Group alerts by drop-down]
 
-Each group displays information such as the alerts' severity and how many users, hosts, and alerts are in the group. The type of information displayed varies depending on the selected field.
+Each group displays information such as the alerts' severity and how many users, hosts, and alerts are in the group. The information displayed varies depending on the selected fields.
```

Comment:
Can we maybe describe how the selection order will determine the nesting order, and that the order is displayed in the "Group alerts by" label?

Reply:
@stephmilovic We do mention the nesting order in the previous paragraph, above the screenshot, but I can add something about how the order is displayed in the label.